docks:
  dock1:
    frame: map
    orientation: [-6.4261442082234274e-06, 6.645387786488029e-06, -0.0034768511841252766,
      0.9999939556919266]
    pose: [-5.10318543132199, -2.8073387240440577, -0.010317167592272834]
    type: simple_dock
